code/modules/mining/ores_coins.dm

```diff
@@ -280,6 +280,16 @@
 	sideslist = list("heads")
 	value = 20
 
+/obj/item/weapon/coin/antagtoken
+	name = "antag token"
+	icon_state = "coin_valid_valid"
+	cmineral = "valid"
+	desc = "A novelty coin that helps the heart know what hard evidence cannot prove."
+	sideslist = list("valid", "salid")

```

Comment:
What is even up with "salid" anywhere in this?
Should be valid salad, shouldn't it?